Doc/distutils/setupscript.rst

```diff
@@ -675,7 +675,7 @@ information is sometimes used to indicate sub-releases.  These are
               'Intended Audience :: Developers',
               'Intended Audience :: System Administrators',
               'License :: OSI Approved :: Python Software Foundation License',
-              'Operating System :: MacOS :: MacOS X',
+              'Operating System :: MacOS :: MacmacOS',
```

Comment:
This one is incorrect for two reasons:

1. Spelling error in the new text
2. This value is a PyPI classifier, and those still use "MacOS X"

Please revert this change until "macOS" is added as an alternatieve to the PyPI classifiers.